config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -116,7 +116,7 @@ presubmits:
     path_alias: k8s.io/kops
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200128-8dae294-experimental
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200128-8dae294-1.15
```

Comment:
we should definitely use these versioned tags for all the jobs that target release branches 👍